releasenotes/notes/Add-qubit-channel-mapping-to-backend-config-schema-7f8f93034074b328.yaml

```
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    Added `qubit_channel_map` and `channel_confiig` to the Pulse enabled backend configuration
```

```suggestion
    Added `qubit_channel_map` and `channel_config` to the Pulse enabled backend configuration
```

Make sure to update this to remove qubit_channel_map 😄